CHANGES.md

```diff
@@ -2,6 +2,11 @@
 
 ## __NEXT__
 
+### Bug Fixes
+
+* filter: Improve help and summary messages. [#1389][] (@victorlin)
```

Comment:
A less vague changelog entry wouldn't be unwelcome… :-)

Reply:
Updated to:

> * filter: Updated the help text of `--include` and `--include-where` to explicitly state that this can add strains that are missing an entry from `--sequences`.
> * filter: Fixed the summary messages to properly reflect force-inclusion of strains that are missing an entry from `--sequences`.
> * filter: Updated wording of summary messages.